### PR TITLE
refactor(eslint): restructure eslint module rules, simplify usage

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

**Review by dshovchko — APPROVED** [review]

File: eslint/src/index.ts [hunk 1]
```diff
@@ -43,7 +24,12 @@ export const configs = {
   },
   'default-5': {
     rules: {
-      ...buildDefault(DEPRECATED_4_RULES, 'error')
+      ...buildDefault(DEPRECATED_5_RULES, 'warn')
```
Comment (abarmina) [inline eslint/src/index.ts:27]: here and below is ‚5’, but what about ‚4’?

**Review by abarmina — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

File: eslint/src/index.ts (hunk 1 above)
Comment (dshovchko) [reply eslint/src/index.ts:27]: We are going to release 5.0.0 so 4 leave in past

Comment (abarmina) [reply eslint/src/index.ts:27]: oh, it’s ‚default-5’, now I see it

**Pull request merged**